__test__/addRemove.test.js

```diff
@@ -0,0 +1,83 @@
+import addTask from '../src/mockFunction/addTask.js';
+import removeTask from '../src/mockFunction/removeTask.js';
```

Comment:
Well done for writing the tests. Kindly ensure that the tests pass.

![image](https://github.com/MariamOsman11/todo-list-test/assets/20554583/f3fcb853-b8d8-4883-a325-d9698640286d)